### ISSUE TITLE
owncloud 9.1.0 Cant't run with php 7.1.0

### ISSUE THREAD

**Issue by cybmp3** [owncloud/core#25858]
owncloud 9.1.0 Cant't run with php 7.1.0.
500 ERROR
Thanks!

 
### Steps to reproduce
1.
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** (see ownCloud admin page)

**Updated from an older ownCloud or fresh install:**

**Where did you install ownCloud from:**

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by DeepDiver1975** [comment]
please have a look at the docs - 7.1 is not supported

**Comment by RealRancor** [comment]
And is btw. still beta...

**Comment by Newcomer1989** [comment]
On starting the OC with PHP 7.1 (is now released - 01 Dec 2016) I get this message:

```
This version of ownCloud is not compatible with PHP 7.1.
You are currently running 7.1.0-5+deb.sury.org~xenial+1. Please use at least ownCloud 9.2.0.
```

I think there is actual no official repository to get the OC 9.2.0. I found only a nightly build, which is not recommend to use (https://download.owncloud.org/download/repositories/nightly/).

I hope this helps for all other ppl, which are also searching for a solution like me... (ehm, the solution is actual -> waiting ;-))

**Comment by kdslkdsaldsal** [comment]
There is no 9.2.0 yet. This gets released as 10.0 in March 2017 and is compatible with PHP 7.1:

https://github.com/owncloud/core/wiki/Maintenance-and-Release-Schedule

**Comment by Studio0815** [comment]
nach 100 versuchen manuel und webinstall etc. etc.
This version of ownCloud is not compatible with PHP 7.1.
You are currently running 7.1.1. Please use at least ownCloud **10.0.0.**
